apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx

```diff
@@ -86,8 +87,12 @@ const Body = ({ data, onClose }: { data: Data; onClose?: () => void | undefined
 
   useEffect(() => {
     if (error) {
-      onCancel(error);
-      if (!(error instanceof WrongDeviceForAccount)) {
+      if (
+        ![
+          error instanceof WrongDeviceForAccount,
+          error instanceof SwapCompleteExchangeError && error.message === "User refused",
```

Comment:
Can we use ErrorStatus instead? libs/ledgerjs/packages/hw-app-exchange/src/ReturnCode.ts

Reply:
Unfortunately we don't have error status at this stage.